tutorials/calibration_flow/static_quant.py

```diff
@@ -11,7 +11,7 @@
 from torchao.dtypes import to_affine_quantized_static
 from torchao.quantization.utils import compute_error
 from torchao.quantization import quantize_
-from torchao.quantization.subclass import to_linear_act_quantized
+from torchao.quantization.subclass import to_linear_activation_quantized
```

Comment:
should this still be from torchao.quantization.subclass?

Reply:
oh right, will update